documentation.html

```diff
@@ -648,6 +650,111 @@ <h1 class="card--heading">Strawberries</h1>
                 </div>
             </div>
             <div class="spacer-8rem"></div>
+
+            <div class="component" id="navbar-section">
+                <!-- There is just one header and header-description per component -->
+                <div class="component__header">
+                    Navbar
+                </div>
+                <div class="component__header-desc">
+                    Navigation bar as the name suggests aids in user navigation to 
+                    important parts of the application or website, generally located at the topmost
+                    part of the website or app.
+                </div>
+               <!-- There could be multiple example illustrations for a particular component -->
+                <div class="component__example-header">
+                    Navbar example
+                </div>
+                <div class="component__example-desc">
+                    Copy the following HTML snippet and edit the navbar
+                    content according to use case.
+                </div>
+                <div class="component__example-demo input__example-demo-container">
+                    <nav>
+                        <div class="img-container">
+                          <img
+                            src="https://mantra-ui.netlify.app/logo_no-padding.jpg"
+                            alt="mantra-logo"
+                          />
+                        </div>
+                        <ul class="nav-links primary-nav-main-links flex">
+                          <li>
+                            <a href="#">Home</a>
+                          </li>
+                          <li>
+                            <a href="#">Products</a>
+                          </li>
+                        </ul>
+                        <ul class="nav-links primary-nav-side-links flex">
+                          <li>
+                            <a
+                              href="#"
+                              class="fab fa-twitter"
+                              ></a
+                            >
+                          </li>
+                          <li>
+                            <a href="#" class="fab fa-github"
+                              ></a
+                            >
+                          </li>
+                        </ul>
+                      </nav>
+                </div>
+                <div class="component__example-code">
```

Comment:
suggestion: don't use this kind of naming for classes, use kebab casing.

Reply:
In this library I have used similar naming throughout the project, but will keep this thing in mind. Thanks for the suggestion.